modul/menu.py

```
@@ -1527,253 +1603,257 @@ def gettext(k):
 
-        indicator_font = pygame.font.Font(None, int(MENU_ITEM_FONT_SIZE * 0.7))
+        indicator_font = pygame.font.Font(None, int(C.MENU_ITEM_FONT_SIZE * 0.7))
 
+        try:
+            from modul.i18n import gettext
+        except Exception:
+            def gettext(key):
+                """Implementation detail: see method body for behavior."""
+                return key
+
+class TTSVoiceMenu(Menu):
```

Pylint (4.0.4)

[error] class already defined line 1133

(E0102)